singlepoll/linux.go

```diff
@@ -177,7 +177,15 @@ func poller(epfd int, evfd *eventfd.EventFD) {
 		// to big and GC overhead increases
 		events := make([]syscall.EpollEvent, 128)
 		n, err := syscall.EpollWait(epfd, events, -1)
-		if err != nil {
+
+		switch err {
+		case nil:
+			// everything is great
+		case syscall.EINTR:
```

Comment:
I don't think that is needed, as it will happen on any signal revived, this is just in case you want to wake up epoll with signal.

It isn't an error per se.